### PR TITLE
Added frameworkAssemblies to nuspec templating

### PR DESCRIPTION
**Pull request by khellang**
japbcvpavbzau9dbuaxf

### CODE REVIEW

File: src/app/FakeLib/NuGet/NugetHelper.fs [hunk 1]
```diff
@@ -19,6 +19,10 @@ type NugetFrameworkReferences =
     { FrameworkVersion : string
       References : NugetReferences }
 
+type NugetFrameworkAssemblyReferences =
```
Comment (khellang) [inline src/app/FakeLib/NuGet/NugetHelper.fs]: I guess I could've reused the existing `NugetFrameworkReferences` type and grouped framework assembly references on `FrameworkVersion` instead. What do you think?

File: src/app/FakeLib/NuGet/NugetHelper.fs [hunk 2]
```diff
@@ -153,9 +159,18 @@ let private createNuSpecFromTemplate parameters (templateNuSpec:FileInfo) =
...
     let referencesXml = sprintf "<references>%s</references>" (references + referencesByFramework)
     
+    let getFrameworkAssemblyTags references =
+        references
+        |> Seq.map (fun x -> sprintf "<frameworkAssembly assemblyName=\"%s\" targetFramework=\"%s\" />" x.AssemblyName x.FrameworkVersion)
+        |> toLines
+
+    let frameworkAssemblies = getFrameworkAssemblyTags parameters.FrameworkAssemblies
+
+    let frameworkAssembliesXml = sprintf "<frameworkAssemblies>%s</frameworkAssemblies>" frameworkAssemblies
```
Comment (forki) [inline src/app/FakeLib/NuGet/NugetHelper.fs]: if frameworkAssemblies = [] then "" else
    sprintf "<frameworkAssemblies>%s</frameworkAssemblies>" frameworkAssemblies

File: src/app/FakeLib/NuGet/NugetHelper.fs (hunk 1 above)
Comment (forki) [inline src/app/FakeLib/NuGet/NugetHelper.fs]: I think it's ok to have both types. We also separate these in paket.

**Comment by forki** [comment]
thanks. only one small issue.

File: src/app/FakeLib/NuGet/NugetHelper.fs (hunk 2 above)
Comment (khellang) [inline src/app/FakeLib/NuGet/NugetHelper.fs]: `frameworkAssemblies` is a `string`. Solved it another way :smile:

File: src/app/FakeLib/NuGet/NugetHelper.fs (hunk 1 above)
Comment (forki) [inline src/app/FakeLib/NuGet/NugetHelper.fs]: I wonder if we should make the FrameworkVersion a string list

**Pull request merged**